test/OpExtInst.ll

```diff
@@ -0,0 +1,126 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: LoopMerge
+; CHECK-SPIRV-NOT: ExtInst
```

Comment:
Such a sequence of checks would fail due to the OpLine bug that cannot be fixed within this patch. I've added a respective comment to the test and will consider creating an issue/investigating myself